docs/sources/mimir/configure/configure-blocked-queries.md

```diff
@@ -30,6 +30,40 @@ To set up runtime overrides, refer to [runtime configuration]({{< relref "./abou
 The order of patterns is preserved, so the first matching pattern will be used.
 {{% /admonition %}}
 
+## Formatting queries to block
+
+Queries received by Mimir are parsed into PromQL expressions before blocking is applied.
+The `pattern` from the blocked queries is compared against Prometheus' string format representation of the parsed query,
+in order to allow consistent query blocking behavior regardless of formatting differences in the submitted queries.
+
+Among other transformations, the Prometheus formatter may reorder operators, remove empty selector braces,
+and eliminate newlines and extraneous whitespace.
+
+You can use Mimirtool's `promql format <query>` command to apply the Prometheus string formatting to a query
+to ensure your provided blocked query `pattern` is correct:
+
+```shell
+$ mimirtool promql format 'foo{}'
+foo  # empty selector braces removed
+```
```

Comment:
Our style guide discourages `$` for shell prompts so that examples can be copy-pasteable. Instead you should use two code blocks, one for the command and one for the expected output.

For more information, refer to https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#command-lines

Additionally, the code blocks comments explaining the behavior should be part of the explanation of the code block and not within it.